crates/fluvio-cluster/src/cli/start/mod.rs

```diff
@@ -101,6 +101,14 @@ pub struct K8Install {
     /// Uses port forwarding for connecting to SC during install
     #[clap(long)]
     use_k8_port_forwarding: bool,
+
+    /// TLS: Client secret name while adding to Kubernetes
+    #[clap(long, default_value = "fluvio-client-tls")]
+    tls_client_secret_name: String,
+
+    /// TLS: Server secret name while adding to Kubernetes
+    #[clap(long, default_value = "fluvio-tls")]
```

Comment:
could you use `TLS_SERVER_SECRET_NAME` instead?

I think that you can use:

```rust
fn default_ tls_server_secret_name() -> &'static str {
    TLS_SERVER_SECRET_NAME
}
```

Reply:
@morenol I am not sure if clap takes function values. We will have to keep it string.

Reply:
ohh, right I said that because I wasn't sure that 

```
    #[clap(long, default_value = TLS_SERVER_SECRET_NAME)]
```

would work

Reply:
Yup. That change i did push :)